src/libserver/busmonitor.cpp

```diff
@@ -101,10 +101,10 @@ A_Busmonitor::send_L_Busmonitor (LBusmonPtr p)
       buf.resize (7);
       EIBSETTYPE (buf, EIB_BUSMONITOR_PACKET_TS);
       buf[2] = p->status;
-      buf[3] = (p->timestamp >> 24) & 0xff;
-      buf[4] = (p->timestamp >> 16) & 0xff;
-      buf[5] = (p->timestamp >> 8) & 0xff;
-      buf[6] = (p->timestamp) & 0xff;
+      buf[3] = p->time_stamp >> 24;
```

Comment:
Please either don't drop the mask on this line, or remove them all.  Any reasonable compiler will know that *all* of these masking ANDs are no-ops anyway. Leaving it off only on the top line looks strange, i.e. when I read that I'd pause to think whether the omission is in any way intentional.

Reply:
Yes, ok. Good point. I've reverted it.